obspy/signal/detrend.py

```diff
@@ -29,6 +31,158 @@ def simple(data):
     return data - (x1 + np.arange(ndat) * (x2 - x1) / float(ndat - 1))
 
 
+def __plotting_helper(data, fit, plot):
+    fig = plt.figure(figsize=(8.0, 5.0))
```

Comment:
`fig, axes = plt.subplots(2, 1, figsize=(8, 5))` and then there's no need for the `gca()`.